### PR TITLE
Migration of eslint-config-spt to schibsted v5.

### PR DESCRIPTION
**Pull request by nosilleg**

### CODE REVIEW

**Comment by SimenB** [comment]
Failing CI :(

**Review by SimenB — APPROVED** [review]
I think this is awesome!

I'd recommend moving to lerna or something, but this is does work, so no rush

**Comment by nosilleg** [comment]
@SimenB CI addressed in forced push. 👍

File: packages/eslint-config-schibsted-chai/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+```bash
+yarn add eslint-config-schibsted eslint-config-schibsted-flow -D
+```
+
+Then add the `extends` option to your `.eslintrc`:
+
+```json
+{
+    "extends": [
+        "spt",
```
Comment (SimenB) [inline packages/eslint-config-schibsted-chai/README.md]: should it be called spt or schibsted?

File: packages/eslint-config-schibsted-chai/chai.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+module.exports = {
+  plugins: ["chai-friendly"],
+  rules: {
+    "no-unused-expressions": 0,
```
Comment (SimenB) [inline packages/eslint-config-schibsted-chai/chai.js]: `"off"`

File: packages/eslint-config-schibsted-chai/chai.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    "chai-friendly/no-unused-expressions": 2
```
Comment (SimenB) [inline packages/eslint-config-schibsted-chai/chai.js]: `"error"`

**Pull request merged**